sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -684,13 +615,13 @@ export class EventHubSender extends LinkEntity {
 
           try {
             await defaultLock.acquire(this.senderLock, () => {
-              return this._init();
+              return this._init(this._createSenderOptions({}));
```

Comment:
Is there any benefit to calling `this._createSenderOptions({})` here? It looks like this gets called in `this._init` if nothing is passed in.

Reply:
Oh, not really - removed it.

This was added initially when I was passing in the `sendTimeoutinSeconds` here as part of the operation timeout for rhea-promise to consider using~